src/shared/cgroup-setup.c

```diff
@@ -674,51 +672,38 @@ int cg_attach_many_everywhere(CGroupMask supported, const char *path, Set* pids,
         return r;
 }
 
-int cg_migrate_everywhere(CGroupMask supported, const char *from, const char *to, cg_migrate_callback_t to_callback, void *userdata) {
+int cg_migrate_controllers(CGroupMask supported, CGroupMask mask, const char *from, cg_migrate_callback_t to_callback, void *userdata) {
```

Comment:
hmm, so this call becomes cgroupvs1-only this way. and there's no indication in name or assert or so.

Reply:
Distinguished the functions by their name.